src/rpc/blockchain.cpp

```diff
@@ -1577,13 +1577,17 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
     BuriedForkDescPushBack(softforks,"bip34", consensusParams.BIP34Height);
     BuriedForkDescPushBack(softforks,"bip66", consensusParams.BIP66Height);
     BuriedForkDescPushBack(softforks,"bip65", consensusParams.BIP65Height);
+    BuriedForkDescPushBack(softforks,"bip147", consensusParams.BIP147Height);
     BuriedForkDescPushBack(softforks, "csv", consensusParams.CSVHeight);
     BuriedForkDescPushBack(softforks, "dip0001", consensusParams.DIP0001Height);
+    BuriedForkDescPushBack(softforks, "dip0003", consensusParams.DIP0003Height);
     BuriedForkDescPushBack(softforks, "dip0008", consensusParams.DIP0008Height);
-    for (int pos = Consensus::DEPLOYMENT_TESTDUMMY + 1; pos != Consensus::MAX_VERSION_BITS_DEPLOYMENTS; ++pos) {
+    BuriedForkDescPushBack(softforks, "dip0020", consensusParams.DIP0020Height);
+    BuriedForkDescPushBack(softforks, "dip0024", consensusParams.DIP0024Height);
+    BuriedForkDescPushBack(softforks, "realloc", consensusParams.BRRHeight);
+    for (int pos = Consensus::DEPLOYMENT_TESTDUMMY; pos != Consensus::MAX_VERSION_BITS_DEPLOYMENTS; ++pos) {
```

Comment:
why? I think the original idea was to keep "testdummy" the last one in the list because it's using the maximum bit possible (28). This part should be reverted imo.

Reply:
```
src/chainparams.cpp:        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].bit = 28;
```
it is still bit number 28.

This loop is going through this enum that is used very limitely:
```
enum DeploymentPos {
    DEPLOYMENT_TESTDUMMY,
    DEPLOYMENT_V19,     // Deployment of Basic BLS, AssetLocks
    DEPLOYMENT_V20,     // Deployment of EHF, LLMQ Randomness Beacon
    // NOTE: Also add new deployments to VersionBitsDeploymentInfo in versionbits.cpp
    MAX_VERSION_BITS_DEPLOYMENTS
};
```

Reply:
yes, but prior to this patch `testdummy` was the _last_ item in `softforks` list (because it was added after the loop) and now it's going to be the _first_ one (because it's the first one in `DeploymentPos`).

Reply:
updated - no more loop at all anymore.